app/datasets.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = {
+    clamp: function () {
+
+        var number = -1,
+            lower = 3,
+            upper = 7;
+
+        return [number, lower, upper];
+    },
+
+    inRange: function () {
+
+        var number = 4,
+            start = 3,
+            end = 6;
+
+        return [number, start, end];
+    },
+
+    random: function () {
+
+        var lower = 4,
+            upper = 7,
+            floating = true;
+
+        return [lower, upper, floating];
+    }
+};
```

Comment:
I think functions in datasets should be empty. User who have to solve this exercise will have to write bodies of those functions.